lib/iris/tests/unit/fileformats/nc_load_rules/actions/test__time_coords.py

```diff
@@ -359,31 +364,17 @@ def test_aux_not_in_phenom_coords(self):
         self.check_result(result, "missing")
 
     def test_aux_fails_typeident(self):
-        # The coord variable is identified as a CFAuxiliaryCoordinate by cf.py,
-        # but having the wrong units causes it to fail the 'is_time' or
-        # 'is_period' test, so the 'provides_coord' rule fails to trigger.
-        # Rules Triggered:
-        #     001 : fc_default
-        #     002 : fc_build_auxiliary_coordinate
-        # Again, though it builds as a 'miscellaneous' rather than a recognised
-        # specific coord type, it makes no practical difference.
-        result = self.run_testcase(
-            coord_dim_name="dim_renamed",
-            dimname="dim_renamed",
-            in_phenomvar_coords=True,
-            units="1",
-        )
-        self.check_result(result, "aux")
-
-    def test_aux_no_coordsref(self):
-        # The coord variable is identified as a CFAuxiliaryCoordinate by cf.py,
-        # but having the wrong units causes it to fail the 'is_time' or
-        # 'is_period' test.
+        # We provide a non-dimension coord variable, identified as a
+        # CFAuxiliaryCoordinate by cf.py, but we also give it the wrong units,
+        # which causes it to fail both 'is_time' and 'is_period' tests, so the
+        # 'provides_coord' rule fails to trigger.
```

Comment:
Is it still true that the 'provides_cord' rule fails to trigger? My understanging is that this used to be the case, but we now handle everything with `action_provides_coordinate` which now "triggers" in a similar way to the other cases.

Also, I'm not sure how clear/accurate it is to describe the units as "wrong", it seems to me that they are simply generic and do not trigger a special rule, unless they contradict some other part of the file that I'm not aware of

Reply:
:+1: yes. good spot.  My logic is wrong, or at least the explanation of how it translates is incorrect.
Will fix !